docs/examples/index.md

```diff
@@ -1,8 +1,113 @@
 ---
-title: Basic usage
-sidebar_label: Basic usage
+sidebar_label: Examples
+title: Examples
 ---
 
-This section contains examples of how to use Apify Client for Javascript.
+# Code examples
 
-There is nothing here yet.
+## Different ways how to pass an input
+
+The fastest way to get results from an Actor is to pass input directly to the `call` function.
+
+```javascript
+import { ApifyClient } from 'apify-client';
+
+// Client initialization with the API token
+const client = new ApifyClient({ token: 'MY_APIFY_TOKEN' });
+
+const actorClient = client.actor('apify/instagram-hashtag-scraper');
+
+// Run the actor and wait for it to finish up to 60 seconds. Input is not persisted for next runs.
+const runData = await actorClient.call({ hashtags: ['twitter'], resultsLimit: 20 }, { waitSecs: 60 });
```

Comment:
Let's split the input and define it separately so it's not confusing for first time users e.g.:

```
const input = { hashtags: ['twitter'], resultsLimit: 20 };
const runData ...
```